### DISCUSSION TITLE
Android build for React Native: Details: "Package not found: app.id"

### DISCUSSION THREAD

**Discussion by rares-lupascu** [codemagic-ci-cd/codemagic-docs#1455] [Q&A] [answered]
Hey

I am trying out codemagic and wanted to do a test with one of our apps
cannot get it to compile on Android 

Build ID: **6405ea74a075e29289551dc6**

Error:
```
Unable to create an edit for package "app.xxx.v1". <HttpError 404 when requesting https://androidpublisher.googleapis.com/androidpublisher/v3/applications/app.xxx.v1/edits?alt=json returned "Package not found: app.xxx.v1.". Details: "Package not found: app.xxx.v1.">
```

any idea how i can get passed this?

thanks

**Comment by himesh-cm** [comment]
Let me check, i will get back to you

**Comment by rares-lupascu** [comment, marked as answer]
hey ... thank you for the reply ... the issue was created by code responsable with calculating the version code number ... i hardcoded it and it worked ... trying now to put it back in to get it working

**Reply by himesh-cm** [reply]
Great, let us know if you hit any roadblocks.

**Reply by rares-lupascu** [reply]
in my case i did not manually add the aab to google play console as part of the setup process ... just to be on the same page ... before any CI/CD stuff ... on google play store u need to manually add the app first so google knows about ur package/bundle identifier ... then it should all be ok
